stdlib/imaplib.pyi

```diff
@@ -157,7 +157,7 @@ class _Authenticator:
     def encode(self, inp: bytes) -> str: ...
     def decode(self, inp: str) -> bytes: ...
 
-def Internaldate2tuple(resp: bytes) -> time.struct_time: ...
+def Internaldate2tuple(resp: ReadableBuffer) -> time.struct_time | None: ...
 def Int2AP(num: int) -> str: ...
```

Comment:
```suggestion
def Int2AP(num: int) -> bytes: ...
```
Not directly related but this one definitely return bytes. We could also change the arg type to `SupportsAbs` I think.

Reply:
Yes, sure! It is `num = int(abs(num))`: `SupportsAbs[SupportsInt]` 😱

Reply:
Still got to fix the return type :)

Reply:
> Yes, sure! It is `num = int(abs(num))`: `SupportsAbs[SupportsInt]` 😱

Could be worse:

https://github.com/python/typeshed/blob/b5c4580d529c03efe7b4546ed9d3632befd5fb49/stdlib/unittest/case.pyi#L219